tests/unit/ec2/test_run_instances.py

```diff
@@ -79,13 +79,15 @@ def test_block_device_mapping(self):
         args_list.append(
             ' [{"DeviceName":"/dev/sda1","Ebs":{"VolumeSize":20}}]')
         result = {
-            'BlockDeviceMapping.1.DeviceName': '/dev/sda1',
-            'BlockDeviceMapping.1.Ebs.VolumeSize': 20,
+            'BlockDeviceMappings': [
+                {'DeviceName': '/dev/sda1',
+                 'Ebs': {'VolumeSize': 20}},
+            ],
             'ImageId': 'ami-foobar',
-            'MaxCount': 1,
-            'MinCount': 1
+            'max_count': 1,
```

Comment:
It accepts both in the Operation.call interface.  See the last paragraph in the PR description for more info.  The plan is to migrate these over eventually.